app/kuma-cp/cmd/run.go

```diff
@@ -117,8 +121,8 @@ func newRunCmdWithOpts(opts runCmdOpts) *cobra.Command {
 					runLog.Error(err, "unable to set up Clusters server")
 					return err
 				}
-				if err := kds_server.SetupServer(rt); err != nil {
-					runLog.Error(err, "unable to set up KDS server")
+				if err := kds_global.SetupServer(rt); err != nil {
```

Comment:
This is Global that connects to Local in order to fetch Dataplanes. I'm planning to implement Local -> Global in the next PR, because Local by default doesn't know about Global and it gets Global IP from the first DiscoveryRequest from Global. 
So it will look like this:
1. Global starts KDSSink for every Local for fetching Dataplanes (this PR)
2. Local start KDSSink for the first request from Global for fetching policies (next PR)

`SetupComponent` sounds good.